### ISSUE TITLE
Compatibility with RN >= 0.48

### ISSUE THREAD

**Issue by flavordaaave** [ptomasroos/react-native-idfa#5]
Since RN 0.48 `AdSupportIOS` and it is recommended to use this plugin (see [here](https://github.com/facebook/react-native/commit/6ad5e2fa7cab74022d1356a705908105e631052c)

But so far this plugin does not seem to be compatible with this RN version.

**Comment by ptomasroos** [comment]
Thanks for the notice. 

Will fix support for rn 0.48 next week. 

Tomas

**Comment by afilp** [comment]
Also, can you please add some instructions on how to install it for both iOS and Android? Thank you!

**Comment by ptomasroos** [comment]
Hey!

I will. I'm currently at the hospital. Will fix in the weekend. 

Sorry. 

Tomas

**Comment by afilp** [comment]
Ok, get well soon!

**Comment by afilp** [comment]
Hi @ptomasroos, we would appreciate if this was fixed. Thanks again!

**Comment by ptomasroos** [comment]
Yes, I'm comitting a fix for you later today @afilp again sorry for delay :(
